src/layout/login/login-form/LoginForm.tsx

```diff
@@ -7,22 +7,54 @@ import { FaFacebook } from 'react-icons/fa';
 import { FcGoogle } from 'react-icons/fc';
 import { loginFormSchema } from '../../../models/schemas.yup';
 import { yupResolver } from '@hookform/resolvers/yup';
+import { useLoginUserMutation } from '../../../api/auth/authApi';
+import { useState } from 'react';
 
-type FormData = yup.InferType<typeof loginFormSchema>;
+const WRONG_EMAIL_OR_PASSWORD = 'Błędny email lub hasło. Spróbuj ponownie.';
+export type LoginSchemaType = yup.InferType<typeof loginFormSchema>;
 
 const LoginForm = () => {
   const {
     register,
     handleSubmit,
+    setError,
     formState: { errors },
-  } = useForm<FormData>({
+  } = useForm<LoginSchemaType>({
     resolver: yupResolver(loginFormSchema),
   });
 
-  const onSubmit = (data: FormData) => {
-    return data;
-  };
+  const navigate = useNavigate();
+  const [isWrongEmailOrPassword, setIsWrongEmailOrPassword] = useState(false);
```

Comment:
`set` bym nazwała bez słowa `is` które jest bardziej dla wartości stanu (po lewej stronie), więc myśle że bardziej `[isWrongEmailOrPassword, setWrongEmailOrPassword]`